tests/conftest.py

```diff
@@ -44,13 +44,14 @@ def reset_backend():
     params=[
         (pyhf.tensor.numpy_backend(), None),
         (pyhf.tensor.pytorch_backend(), None),
+        (pyhf.tensor.pytorch_backend(float='float64', int='int64'), None),
         (pyhf.tensor.tensorflow_backend(session=tf.compat.v1.Session()), None),
         (
             pyhf.tensor.numpy_backend(poisson_from_normal=True),
             pyhf.optimize.minuit_optimizer(),
         ),
     ],
-    ids=['numpy', 'pytorch', 'tensorflow', 'numpy_minuit'],
+    ids=['numpy', 'pytorch', 'pytorch64', 'tensorflow', 'numpy_minuit'],
```

Comment:
Maybe I'm missing something obvious, but why don't we also have a `'tensorflow64'` opetion here?

Reply:
we could add it, just worried it would increase testing time at some pooint?

Reply:
Yeah, it definitely will. It just seems a bit strange to only have one of the backends be tested at 64. I'll approve for now and let @kratsg give further thoughts.

Reply:
i would rather have this split up into 64b backends and 32b backend parameterizations (using `backend32` and `backend64`). Since it's clear we really need to be concerned about consistency, we should be keeping things on equal footing.

Reply:
hm, but then do we need separate tests that request the backends64 fixture?

Reply:
I've been thinking about this for a while. I suspect that we will be able to test to a tighter precision with 64b than we can with 32b. In which case, I would want to see stronger tests for the 64b backend than the 32b. This could just be parameterized by adding information about the precision along with the backend?

Reply:
yes we can absolutely compare numpy-torch-tf at higher precision when they're all at double precision (and perhaps even use the same optimizer)